gateway/1.6/modules/ROOT/pages/policies-included-traffic-management-route.adoc

```diff
@@ -0,0 +1,175 @@
+= Traffic Management for Multiple Upstream Services
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+:keywords: route, traffic management, upstream service
+
+[%autowidth.spread,cols="a,a"]
+|===
+>s| Policy Name | Traffic Management for Multiple Upstream Services
```

Comment:
(what does the `>s` do?)

Reply:
It bolds the text in that first column. FWIW it's here because all our legacy policies have this table formatting, and I included it for consistency.